sample.conf

```diff
@@ -476,10 +505,13 @@ HideURL		*.ra
 # to define it's search terms.
 
 SearchEngine	yahoo.com	p=
+SearchEngine	altavista.com	q=
```

Comment:
`altavista.com` is gone - it's `search.yahoo.com` now. `msn.com` redirects to `bing.com`. `eureka.com` is just gone.

Unless you are you thinking that these are required for processing historical logs. I hadn't thought of that. Maybe this deserves a comment to explain that while they no longer exist, having them in the configuration might be useful for this purpose.

Reply:
I have removed them for now, but keep them in my "modular config". If you think they are really helpful for historical reasons, they can always be added back with a proper hint.